catalog.css

```diff
@@ -0,0 +1,309 @@
+body {
+  margin: 0;
+  padding: 0;
+
+  font-family: "RobotoRegular", Helvetica, sans-serif;
+  font-size: 16px;
+  line-height: 18px;
+
+  color: #283136;
+}
+
+.visually-hidden {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  margin: -1px;
+  border: 0;
+  padding: 0;
+  clip: rect(0 0 0 0);
+  overflow: hidden;
+}
+
+header {
+  background-color: #eeeeee;
+}
+.header-logo:hover,
+.header-logo:focus {
+  opacity: 0.8;
+}
+.header-logo:active {
+  opacity: 0.3;
+}
+
+.header-navigation ul {
+  list-style: none;
+}
+.header-navigation a,
+.cart a {
+  font-family: "RobotoMedium", Helvetica, sans-serif;
+  font-size: 16px;
+  line-height: 30px;
+
+  text-decoration: none;
+  text-transform: uppercase;
+
+  color: black;
+}
+header h1 {

```

Comment:
where is your class for header? 
You can have 33 headers on your web site. it's a global rule. Useless global rules are shitty.